.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: ci
+
+on: [pull_request, push]
```

Comment:
I don't think we want this running on every push.

```suggestion
on: [pull_request]
```

Reply:
Done